.travis.yml

```
@@ -50,6 +50,35 @@ matrix:
             - git
             - build-essential
       env: SUITE=ci-all-features
+    - rust: 1.33.0
```

Please give this job a name

```suggestion
    - name: Code Coverage
      rust: 1.33.0
```